less/v2/chat.less

```
@@ -653,6 +654,14 @@ enchantment-application {
       }
     }
   }
+
+  .count-area {
+    margin-block-start: 2px;
+    margin-inline-end: 4px;
+    font-size: var(--font-size-10);
+    text-align: end;
+    font-variant: small-caps;
```

This makes the text the same height as the numbers which I think looks a little better, and is a little easier to read given that it's quite small already at size 10.

```suggestion
    text-transform: uppercase;
```